zmsstatistic/src/Zmsstatistic/Download/RequestReport.php

```
@@ -135,8 +135,21 @@ public function writeReportData(ReportEntity $report, \PhpOffice\PhpSpreadsheet\
+        )
+             && is_numeric($report->data['average_processingtime_overall'])
+             ? ReportHelper::formatTimeValue($report->data['average_processingtime_overall']) : '0';
         $sumRowIndex = $sheet->getHighestRow() + 2;
-        $sumRow = array_merge(['Summe', '', $totalSum], $dateSums);
+        $sumRow = array_merge(
+            [
+                'Summe',
+                $overallProcessingTime,
+                $totalSum
+            ]
+            $dateSums
```

PHPStan (2.2.7)

[error] Syntax error, unexpected T_VARIABLE, expecting ')' on line 151


(phpstan.parse)